index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@100&family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+    <header>My Logo</header>
+    <div class="menu-img"><img src="images/Union.svg"></div>
+    </div>
+    <section id="first-sec">
+        <h1>I'm Tebe <br> Glad to see <br> you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3>LET'S CONNECT</h3>
+        <ul class="social-img">
+            <li><a href=""><img src="images/Vector (3).svg"></></li>
+            <li><a href=""><img src="images/Linkedin icon.svg"></></li>
+            <li><a href=""><img src="images/Vector (2).svg"><a></li>
+            <li><a href=""><img src="images/Vector (1).svg"></a></li>
+            <li><a href=""><img src="images/Vector.svg"></a></li>
+        </ul>
+    </section>
+    <section id="second-sec">
+        <div class="snapshoot-img-1"><img src="images/Snapshoot Portfolio (3).png"></div>
+        <h2>Tonic</h2>
+        <ul class="snapshoot">
+            <li>CANOPY</li>
+            <li><img src="images/Counter.png"></li>
+            <li>Back END DEV</li>
+            <li><img src="images/Counter.png"></li>
+            <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tonic">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+        <button>See Project</button>
+    </section>
```

Comment:
- [ ] Kindly remove this section as it is not part of what is required for this milestone.